tests/test_interface.rb

```diff
@@ -40,7 +40,8 @@ class TestInterface < CiscoTestCase
   DEFAULT_IF_IP_PROXY_ARP = false
   DEFAULT_IF_IP_REDIRECTS = true
   DEFAULT_IF_VRF = ''
-  IF_DESCRIPTION_SIZE = 243 # SIZE = VSH Max 255 - "description " keyword
+  IF_DESCRIPTION_VALID_SIZE = 68 # SIZE = VSH Max 80 - "description " keyword
+  IF_DESCRIPTION_INVALID_SIZE = 255
```

Comment:
Ah I see what you mean now.  The valid test fills the description command with garbage characters up to the accepted limit.  Since the keyword `description` is included in the command, VALID_SIZE is equal to `80 - length("description ")`.  This is sort of explained in [this comment](https://github.com/cisco/cisco-network-node-utils/pull/179/files#diff-cf2422f8b2cb8aa033a2f17f39d7354fR43).

I agree it's not very obvious why we need the two different values.  Instead of using constants, I could add a `description_max` getter that explicitly performs some of this logic:

```
def description_max
  max_vsh = node.product_id =~ /N(5|6)/ ? 80 : 255
  max_vsh - 'description '.length
end